laravel/app/Console/Commands/InitiateEval.php

```diff
@@ -98,8 +98,8 @@ public function initiateEvaluations()
 		$yesterday = strtotime ( '-1 day' , strtotime ( $date ) ) ;
 		$yesterday = date ( 'Y-m-d' , $yesterday );
 		
-		$residentId = DB::table('evaluation_data')->where('date', $yesterday)->pluck('rId');
-		$attendingId = DB::table('evaluation_data')->where('date', $yesterday)->pluck('aId');	
+		$residentId = DB::table('evaluation_data')->where('date', $yesterday)->pluck('resident_id');
+		$attendingId = DB::table('evaluation_data')->where('date', $yesterday)->pluck('attending_id');	
 		$evalID = DB::table('evaluation_data')->where('date', $yesterday)->pluck('id');			
```

Comment:
I'll comment here since github won't let comment on the actual line.
I think `Service` should be changed to `evaluation_forms_id` line 128

`$evaluationFormsId = Rotations::where('Name', $evaluateeName)->where('Start', $evaluateeRotationsStart[$j])->value('Service');`

Reply:
Good catch!